scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -27,14 +27,18 @@ Specifically, we'll have a look at four samples from the [`SCPCP000005` project]
 The particular libraries we'll integrate come from two rhabdomyosarcoma (RMS) patients, with two samples from each of two patients, all sequenced with 10Xv3 technology.
 Each library is from a separate biological sample.
 
-We'll be integrating these samples with two different tools, [`fastMNN`](http://www.bioconductor.org/packages/release/bioc/html/batchelor.html) ([Haghverdi _et al._ 2018](https://doi.org/10.1038/nbt.4091)) and [`harmony`](https://portals.broadinstitute.org/harmony/) ([Korsunsky _et al._ 2019](https://doi.org/10.1038/s41592-019-0619-0)). 
+We'll be integrating these samples with two different tools, [`fastMNN`](http://www.bioconductor.org/packages/3.16/bioc/html/batchelor.html) ([Haghverdi _et al._ 2018](https://doi.org/10.1038/nbt.4091)) and [`harmony`](https://portals.broadinstitute.org/harmony/) ([Korsunsky _et al._ 2019](https://doi.org/10.1038/s41592-019-0619-0)). 
 Integration corrects for batch effects that arise from different library preparations, genetic backgrounds, and other sample-specific factors, so that datasets can be jointly analyzed at the cell level. 
 `fastMNN` corrects for batch effects using a faster variant of the mutual-nearest neighbors algorithm, the technical details of which you can learn more about from this [vignette by Lun (2019)](https://marionilab.github.io/FurtherMNN2018/theory/description.html).
 `harmony`, on the other hand, corrects for batch effects using an iterative clustering approach, and unlike `fastMNN`, it is also able to consider additional covariates beyond just the batch groupings.
 
 Regardless of which integration tool is used, the `SingleCellExperiment` (SCE) objects first need to be reformatted and merged into a single (uncorrected!) SCE object that contains all cells from all samples.
 This merged SCE can then be used for integration to obtain a formally batch-corrected SCE object.
 
+![Single-cell roadmap: Integration Overview](diagrams/roadmap_multi_merge-integrate.png)
```

Comment:
With this change, the other part of my comment seems to have been hidden on this page, so I am adding it again here:

> In conjunction with this diagram (either version), I think it will be worth adding a note about what processing has already been done on these samples. i.e., that they have already been filtered and normalized.

I also think we could now move this diagram up higher in this notebook...after the first paragraph probably makes sense, and that would be a good place to also add a bit more text about ScPCA processing that has been done.